OpenRA.Mods.RA/Scripting/Global/ReinforcementsGlobal.cs

```diff
@@ -110,7 +110,7 @@ public Actor[] Reinforce(Player owner, string[] actorTypes, CPos[] entryPath, in
 			"unless a custom exitFunc has been supplied. actionFunc will be called as " +
 			"actionFunc(Actor transport, Actor[] cargo). exitFunc will be called as exitFunc(Actor transport).")]
 		public LuaTable ReinforceWithTransport(Player owner, string actorType, string[] cargoTypes, CPos[] entryPath, CPos[] exitPath = null,
-			LuaFunction actionFunc = null, LuaFunction exitFunc = null)
+			int waitLand = 0, int waitReturn = 25, LuaFunction actionFunc = null, LuaFunction exitFunc = null)
```

Comment:
I have to admit I also dislike this huge overload and the API change, but hard-coded values are also bad.